### PR TITLE
Galaxy Dust Mass Functions

### PR DESCRIPTION
**Pull request by wmfw23**
I'm adding the observational data I've collated in this fork, so far just dust mass function data from Beeston+18 and Pozzi+20.

### CODE REVIEW

**Comment by JBorrow** [comment]
Looks like you have addressed a couple of my comments already - the only stuff left is the GSMF -> GDMF(?)

File: data/GalaxyDustMassFunction/conversion/convertBeeston2018.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+    data = np.genfromtxt(raw_file_name, comments="#")
+
+    # array of the lower redshift bin edges for each GDMF
+    z_bins_arr = np.unique(data[:, -2])
+
+    gsmf_arr = []
+    for zlow in z_bins_arr:
+        bdx = data[:, -2] == zlow
+        gsmf_arr.append(data[bdx, :-2])
```
Comment (JBorrow) [inline data/GalaxyDustMassFunction/conversion/convertBeeston2018.py]: gsmf still present in variable names

**Comment by wmfw23** [comment]
Ah didn't see your comments on the last pull request, just tried to nuke it because I had system specific problems with mpi4py etc which were just added to try and debug (and i'm a git novice). Glad I noticed the same changes. gsmf should have been eradicated irrespective of case now.

**Comment by JBorrow** [comment]
Thanks James!

**Pull request merged**